src/NUnitFramework/tests/Internal/Results/TestResultFailureTests.cs

```diff
@@ -202,7 +202,7 @@ public void SuiteResultXmlNodeIsFailure()
             Assert.NotNull(messageNode, "No <message> element found");
             Assert.AreEqual(TestResult.CHILD_ERRORS_MESSAGE, messageNode.Value);
 
-            TNode stacktraceNode = failureNode.SelectSingleNode("stacktrace");
+            TNode stacktraceNode = failureNode.SelectSingleNode("stack trace");
```

Comment:
I know we have our own code representing XML, but we cannot have a element with a space in the name.

Reply:
Thanks! I'm rereviewing too. I don't intend to make any behavioral changes. I'll blame almost programming for 21 hours straight. 🤨